src/components/Cohort/Preview/Preview.tsx

```diff
@@ -116,9 +117,11 @@ const Preview: React.FC<PreviewProps> = ({
   loading
 }) => {
   const classes = useStyles()
+  const location = useLocation()
   const title = group.name
 
   const { vitalStatusData, genderData } = getGenderRepartitionSimpleData(genderRepartitionMap)
+  const isExploringCohort = location.pathname.split('/')[1] === 'cohort'
```

Comment:
can't we use exploredCohort.cohortType (which does not exist yet, maybe add it to the state ?)